myuw/data/resource_link_import.csv

```diff
@@ -1,154 +1,168 @@
+Libraries,Library Resources,all,http://hsl.uw.edu,Health Sciences Library,,,,,,,no
+Student Finances,Tuition and Financial Aid,all,http://finance.uw.edu/sfs/home,Student Fiscal Services,,,,,,,no
+Student Finances,Tuition and Financial Aid,all,,,http://www.washington.edu/students/osfa/,Financial Aid and Scholarships,http://www.bothell.washington.edu/financialaid,Financial Aid and Scholarships,http://www.tacoma.uw.edu/uwt/financial-aid,Financial Aid and Scholarships,no
+Student Finances,Tuition and Financial Aid,all,https://sdb.admin.uw.edu/sisStudents/uwnetid/finaidstatus.aspx,Financial Aid Status,,,,,,,no
+Student Finances,Tuition and Financial Aid,all,http://www.lib.washington.edu/commons/services/gfis,Graduate Funding Information Service (GFIS),,,,,,,no
+Student Finances,Taxes,all,https://sdb.admin.uw.edu/sisStudents/uwnetid/irs1098t.aspx,Form 1098-T,,,,,,,no
+Student Finances,Taxes,all,http://f2.washington.edu/fm/sfs/tax,Student Tax Information,,,,,,,no
+Student Finances,Tuition and Financial Aid,all,http://f2.washington.edu/fm/sfs/tuition,Tuition Overview,,,,,,,no
+Teaching,Course Evaluation,all,https:washington.edu/cec,Course Evaluation Catalog,,,,,,,no
+Teaching,Course Evaluation,all,,,https://uw.iasystem.org/faculty,My Course Evaluations,https://uwb.iasystem.org/faculty,My Course Evaluations,https://uwt.iasystem.org/faculty,My Course Evaluations,no
+Teaching,Course Evaluation,all,,,https://www.washington.edu/assessment/course-evaluations/,Course Evaluations,https://www.washington.edu/assessment/course-evaluations/,Course Evaluations,https://www.tacoma.uw.edu/teaching-learning-technology/iasystem-course-evaluations,Course Evaluations,no
+Teaching,Course Materials,all,https://www.ubookstore.com/faculty,Order Textbooks,,,,,,,no
+Teaching,Course Materials,all,https://finance.uw.edu/c2/printing-copying/course-packs,Request Course Packs,,,,,,,no
+Teaching,Course Materials,all,https://lib.washington.edu/types/course/instructors/request,Request Course Reserves,,,,,,,no
+Teaching,Grading,all,https://gradepage.uw.edu,GradePage,,,,,,,no
+Teaching,Grading,all,https://apps.registrar.washington.edu/grade-change/pages/change.php,Change Submitted Grades,,,,,,,no
+Teaching,Grading,all,,,http://depts.washington.edu/grading,Faculty Resource on Grading,https://www.uwb.edu/registration/policies/grading,Grading Policies,https://www.tacoma.uw.edu/uwt/registrar/grading-policy,Grading Policy,no
+Teaching,Tools,all,https://canvas.uw.edu,Canvas LMS,,,,,,,no
+Teaching,Tools,all,https://catalyst.uw.edu/,Catalyst Web Tools,,,,,,,no
+Teaching,Tools,all,https://uw.hosted.panopto.com,Panopto Lecture Capture,,,,,,,no
+Teaching,Tools,all,https://itconnect.uw.edu/learn/tools/polleverywhere/,Poll Everywhere,,,,,,,no
+Teaching,Tools,all,https://coda.uw.edu,Course Stats,,,,,,,no
+Teaching,Tools,all,https://washington.zoom.us,Zoom,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/,UW Research Website,,,,,,,no

```

Comment:
We don't typically add "website" to site titles. I suggest it just be "UW Research"

Reply:
Updated to "UW Research" !